### ISSUE TITLE
Familiarisation with Invenio Jobs UI

### ISSUE THREAD

**Issue by richard-jones** [CottageLabs/invenio-notify#7]
Invenio has a new Jobs UI, and we will require this to help us process incoming notifications.  At this stage we need to familiarise ourselves with this new Jobs interface, and convince ourselves that we can:

* Use the jobs system to run asynchronous processing of notifications
* Use the Jobs UI to control the active/inactive status of the notification processing for the purposes of controlling incoming notifications

# Acceptance Criteria

*List the criteria which must be met for the issue to be considered complete*

* We understand the Jobs UI and the way in which we can use it in the notify integration

**Comment by philipkcl** [comment]
in new PR, user can create notify job from UI, but schedule / cron setting seem not working.

**Comment by richard-jones** [comment]
@Steven-Eardley @J4bbi could you talk to the invenio technical team about the cron setting, see if there's an issue or something we are missing?

**Comment by richard-jones** [comment]
Agreed today to try this out in a production-like environment (our test machine), and see if we can get it working ourselves first.
